source/agora/common/Config.d

```diff
@@ -560,3 +574,79 @@ unittest
     auto scp_quorum = toSCPQuorumSet(quorum);
     assert(scp_quorum.toQuorumConfig() == quorum);
 }
+
+/*******************************************************************************
+
+    Parse the `event_handlers` config section
+
+    Params:
+        ptr = pointer to the Yaml node containing the loggers configuration
+        c = the parsed command line arguments, for override
+
+    Returns:
+        the parsed event handlers
+
+*******************************************************************************/
+
+private EventHandlerConfig parserEventHandlers (Node* node, const ref CommandLine c)
+{
+    if (node is null)
+        return EventHandlerConfig.init;
+
+    const(string)[] parseSequence (Node node, string section)
+    {
+        if (auto elem = section in node)
+            enforce(node[section].type == NodeType.sequence,
+                format("`%s` section must be a sequence", section));
+        else 
+            return null;
+
+        string[] result;
+        foreach (string item; node[section])
+            result ~= item;
+
+        return result;
```

Comment:
The double lookup is unnecessary.

```suggestion
        auto elem = section in node;
        if (elem is null)
            return null;

        enforce(elem.type == NodeType.sequence,
            format("`%s` section must be a sequence", section));

        string[] result;
        foreach (string item; *elem)
            result ~= item;

        return result;
```

Reply:
`node[section]`,I can get rid of this. That's a good point.